src/doc/rustc/src/platform-support.md

```diff
@@ -118,6 +118,7 @@ The `std` column in the table below has the following meanings:
 
 * ✓ indicates the full standard library is available.
 * \* indicates the target only supports [`no_std`] development.
+* ? indicates the standard library support is unknown or a work-in-progress.
```

Comment:
For reference, the wording here was suggested on zulip: https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/Std.20status.20for.20UEFI/near/419113012

Only question I have is whether there are really targets where the support is unknown? I would expect that since the targets are implemented in this repo we would know whether they support std or not, but I don't know for sure.

Reply:
Nothing should ever be unknown except when it's badly documented (which a lot of this stuff here is!), so especially when starting to migrate #120745, a lot of things will probably be unknown and someone will have to figure it out.

Reply:
Thanks, makes sense :)